website/docs/examples/controller_track/controller_track.md

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
Rename file to README.md

Reply:
Keep it consistent with the other examples

Reply:
Fixed in commit https://github.com/farm-ng/amiga-dev-kit/pull/208/commits/069b0562d8ada0b94579834f7c45f2d2bfc95692